src/pages/books.tsx

```diff
@@ -0,0 +1,389 @@
+        <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="156"
+            height="42"
+            viewBox="0 0 156 42"
+            fill="none"
+        >
+            <g clipPath="url(#clip0_156_755)">
+                <path
+                    d="M12.4999 4.8045C10.8613 2.89724 9.44858 0.948152 9.16209 0.54788C9.13632 0.512035 9.08176 0.512035 9.05599 0.54788C8.7695 0.948152 7.35676 2.89724 5.71817 4.80301C-8.33189 22.4403 7.9358 34.3395 7.9358 34.3395L8.25867 34.5605L8.18136 33.1267L9.06357 14.1586C9.0666 14.1004 9.153 14.1004 9.15603 14.1586L10.0382 33.1267L9.96244 34.5859L10.2838 34.338C10.2823 34.3395 26.5515 22.4403 12.4999 4.8045Z"
+                    fill="white"
+                />
+                <path
+                    d="M10.0848 33.0431C10.0848 33.2626 9.10857 34.0811 9.10857 34.0811C9.10857 34.0811 8.13239 33.2611 8.13239 33.0431C8.13239 32.9848 8.00657 33.4194 8.04447 34.09C8.14906 35.9241 8.49618 38.9172 8.49618 38.9172H9.10857H9.72096C9.72096 38.9172 10.0681 35.9241 10.1727 34.09C10.2106 33.4194 10.0848 32.9848 10.0848 33.0431Z"
+                    fill="white"
+                />
+            </g>
+            <defs>
+                <clipPath id="clip0_156_755">
+                    <rect
+                        width="155.227"
+                        height="41.1906"
+                        fill="white"
+                        transform="translate(0 0.520996)"
+                    />
+                </clipPath>
+            </defs>
+        </svg>
+    </div>
+);
+
+const RelatedBookCard: React.FunctionComponent<Book> = ({
+    title,
+    authors,
+    thumbnail,
+}) => (
+    <div
+        sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            minHeight: '300px',
+            alignItems: 'start',
+            textAlign: 'left',
+        }}
+    >
+        <div
+            sx={{
+                boxSizing: 'content-box',
+                flexShrink: 0,
+                position: 'relative',
+                height: '256px',
+            }}
+        >
+            <img
+                alt={`${title} thumbnail`}
+                src={thumbnail}
+                sx={{ height: '100%' }}
+            />
+        </div>
+        <TypographyScale
+            variant="body3"
+            sx={{
+                color: '#fff',
+                margin: '24px 0 4px 0',
+                fontWeight: '700',
+                fontSize: ['14px, null, null, 16px'],
+            }}
+        >
+            {title}
+        </TypographyScale>
+        <TypographyScale
+            variant="body4"
+            sx={{ color: '#fff', fontSize: ['14px, null, null, 16px'] }}
+        >
+            by {authors.join(', ')}
+        </TypographyScale>
+    </div>
+);
+
+const FeaturedBookCard: React.FunctionComponent<FeaturedBook> = ({
+    title,
+    authors,
+    thumbnail,
+    publishedDate,
+    description,
+    amazonLink,
+    packtLink,
+}) => (
+    <div
```

Comment:
No SEO section ?

Reply:
ah right, there was nothing specified so i left it out but I will put something in.